Add ServiceWorker.postMessage (#24494)
mdn/content: files/en-us/web/api/worker/postmessage/index.md

{{APIRef("Web Workers API")}}

> The **`postMessage()`** method of the {{domxref("Worker")}} interface sends a message to the worker's inner scope. The first parameter is the data to send to the worker. The data may be any value or JavaScript object handled by the [structured clone](/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm) algorithm, which includes cyclical references.

Reviewer: ```suggestion
The **`postMessage()`** method of the {{domxref("Worker")}} interface sends a message to the worker. The first parameter is the data to send to the worker. The data may be any JavaScript object that can be handled by the [structured clone algorithm](/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm).
```

(again, unless "value or" or "inner scope" are important.)
